core/lib/spree/testing_support/factories/shipping_method_factory.rb

```diff
@@ -10,6 +10,12 @@
       end
     end
 
+    after(:create) do |shipping_method, evaluator|
+      if shipping_method.stock_locations.empty?
+        shipping_method.stock_locations << (Spree::StockLocation.all.to_a)
```

Comment:
is this necessary? We've had a lot of issues with factories which depend on the state of the database when they are created